Gruntfile.js

```diff
@@ -143,7 +147,7 @@ module.exports = function( grunt ) {
           cwd: 'svg-min-react/',
           src: [ '**/*.svg' ],
           filter: 'isFile',
-          dest: 'build/'
+          dest: 'react-icons/'
```

Comment:
Nitpick: We don't suffix our other build directories with `-icons` (since they all are icons, by definition 😉). So let's simply call it `react` or `jsx`.